hw5/tasks/task8.py

```diff
@@ -0,0 +1,36 @@
+"""
+Задача 8: Перехват ошибок +1
+
+Напишите программу, которая ожидает пользовательский ввод с клавиатуры двух
+чисел, делит первое на второе соответственно при помощи целочисленного деления
+и выводит результат.
+
+При попытке деления на 0 нужно перехватить ошибку и распечатать строку 'Деление
+на 0 невозможно'.
+
+> При обработке исключения нужно явно указать тип ошибки деления на 0.
+
+Ввод:
+1
+0
+
+Вывод:
+Деление на 0 невозможно
+
+
+Ввод:
+1
+2
+
+Вывод:
+0
+"""
+
+number1 = int(input("Введите число 1: "))
+number2 = int(input("Введите число 2: "))
+
+try:
+    result = number1 // number2
```

Comment:
Здесь также можно в одну строчку:

```suggestion
try:
    result = int(input()) // int(input())
```